tests/test_loss_masking.py

```diff
@@ -22,57 +24,25 @@ def create_tok_fixtures():
     return special_token_left, special_token_individual, batched_tokens
 
 
-def test_create_replace_before_tok():
+def test_target_mask_left():
+    args, _, _, _, _, _ = create_train_fixtures()
     special_token, _, batched_tokens = create_tok_fixtures()
```

Comment:
nit: I would actually just copy paste create_tok_fixtures here - it's duplicated but makes the tests easier to read and insepct.